docs/v0.2.0/index.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Home"
+layout: default
+---
+
+# Installation
+
+[![NuGet Badge](https://buildstats.info/nuget/Arcus.Messaging.Abstractions?packageVersion=0.1.0)](https://www.nuget.org/packages/Arcus.Messaging.Abstractions/0.1.0)
+
+The features are available on NuGet:
+
+```shell
+PM > Install-Package Arcus.Messaging.* -Version 0.1.0
+```
+
+# Features
+
+- Support for using message pumps for the following brokers:
+    - Azure Service Bus ([docs](features/message-pumps/service-bus))
+    - Azure Service Bus Extensions ([docs](features/service-bus))
+- Support for exposing TCP health probes to periodically check liveness/readiness of the host ([docs](features/tcp-health-probe))
+- Customize message pumps ([docs](features/message-pumps/customization))
+
+## Older versions
+
+- [v0.1.0](v0.1.0)
```

Comment:
This path has to be a path higher instead, otherwise one will end up with `/v0.2.0/v0.1.0`.

Reply:
Good catch, thanks!